audio/src/main/java/com/voxeet/audio2/manager/BluetoothHeadsetDeviceManager.java

```diff
@@ -329,10 +331,15 @@ private void closeBluetoothDevices(@Nullable Boolean /*unused*/ disconnect) {
                     need_disconnection = true;
                     break;
                 default:
+
             }
 
             if (need_disconnection) {
+                String id = __Opt.of(device.bluetoothDevice())
+                        .then(android.bluetooth.BluetoothDevice::getAddress).or("");
+                PlatformDeviceConnectionWrapper wrapper = wrappers.get(id);
                 try {
+                    wrapper.setPlatformConnectionState(ConnectionState.DISCONNECTED);
```

Comment:
A safer call should be to check if the wrapper is not null however this case should never be happening